### ISSUE TITLE
Old localized URL is not redirecting to current

### ISSUE THREAD

**Issue by aswanson-nr** [newrelic/docs-website#3490]
### Issue

The `https://docs.newrelic.co.jp/` URL does not point to a valid domain. 

### Some approaches

1. Solve it with our own normal redirects
 >  I'm not sure that this is possible since the meta-redirects plugin seems to just work on relative urls

2. Submit a feature request to gatsby
 > if it comes to this JV can reach out.

3. Move back to Amplify or use Netlify under the hood
 > This is also not an option I would like to choose

### Expected behavior 
The domain should redirect to `https://docs.newrelic.com/jp/`.  to do this we need to:

- [ ]     Figure out how to redirect requests from a certain domain to another domain at a particular path
- [ ]     Update the CNAME for https://docs.newrelic.co.jp/ to point to the docs site on its current hosting platform
- [ ]     Contact the marketing team hero in #help-website and ask them to change any link references for  `docs.newrelic.co.jp` TO  `https://docs.newrelic.com/jp/` so we can ensure this JP page is pointing directly to our new jp destination for docs.

**Comment by aswanson-nr** [comment]
@jpvajda here's an update:

The `https://docs.newrelic.co.jp/` domain was forgotten during migration, so it does not currently point to anything. This is an old address for localized content, so broken links shouldn't be wide spread. While the site was running on Amplify, we had a redirect rule setup through Amplify to automatically send requests for `https://docs.newrelic.co.jp/` to `https://docs.newrelic.com/jp/`. Unfortunately, our current hosting provider does not offer this type of feature.

Things to do:
* Figure out how to redirect requests from a certain domain to another domain at a particular path
* Update the CNAME for `https://docs.newrelic.co.jp/` to point to the docs site on its current hosting platform

**Comment by jpvajda** [comment]
@aswanson-nr what's the actually count of redirects from `https://docs.newrelic.co.jp/` do we have any idea? I'm opting we just let this redirect go at this point instead of spending time on how to figure out to set it up I'd imagine the amount of redirects is low, but I'm not sure how to validate that assumption.

Perhaps @austin-schaefer has some thoughts here.

**Comment by aswanson-nr** [comment]
It's a weird one, since it isn't like our other redirects. Our normal ones just change the url path to another path on the same domain. This one existed outside of our actual site and just handled `.co.jp` to `.com/jp`. I took a crack at checking our PageView events to see if it was being used and I'm not seeing any requests for that domain in the last 5 months, although there are a 400 or so requests if you extend the time range to 6 months back.
Here is my query 
```SELECT count(domain) from PageView facet domain where appName = 'docs.newrelic.com' LIMIT MAX SINCE 5 months ago```

Even if we don't do a redirect, we probably want to update the dns entry to point to the main docs site.

**Comment by jpvajda** [comment]
@aswanson-nr good point, and thanks for the data pull. 📊    if we need to redirect it anyway, we might as well direct it to the jp site url. How much effort do you think this is?

**Comment by aswanson-nr** [comment]
That's the difficult part, still waiting on some internal folks to get back to me on whether or not this can just be configured on their end. If it can't, we would have a few options:
1. Solve it with our own normal redirects 
    * I'm not sure that this is possible since the meta-redirects plugin seems to just work on relative urls
1. Submit a feature request to gatsby
1. Move back to Amplify or use Netlify under the hood

**Comment by jpvajda** [comment]
@aswanson-nr   

`1` --> let's see if we can do some research to see if that is possible.
`2` --> if it comes to that, I can reach out to Gastby
`3` --> 🤣  we won't do this 😉

**Comment by jpvajda** [comment]
after discussions with the team bumping to a P1 and added some specific AC.

**Comment by jpvajda** [comment]
To add some context on the increase in priority:  the entry point from  https://docs.newrelic.co.jp/ was the primarily URL being used in sites like docs.newrelic.co.jp  to point users to to localized version of docs.newrelic.com. So now with this being broken, it's impacted a lot of users in Japan that either have bookmarked that URL or visit docs.newrelic.co.jp  and try to go our Docs.

**Comment by aswanson-nr** [comment]
`docs.newrelic.co.jp` now points directly to the english site. The current workaround is to then swap the language.
We are looking into options to handle the missing redirect.

**Comment by jpvajda** [comment]
@aswanson-nr Nice. I created a follow up issue for next steps here: https://github.com/newrelic/docs-website/issues/3542
